openwisp_radius/admin.py

```diff
@@ -633,7 +634,10 @@ def has_delete_permission(self, request, obj=None):
 
     Token = apps.get_model('authtoken', 'Token')
```

Comment:
Hi @devkapilbansal I think we need to change this to `Token = apps.get_model('authtoken', 'TokenProxy')`, because recently DRF was updated to 3.12.x. See this:- https://github.com/openwisp/openwisp-radius/pull/203/files